src/Sylius/Bundle/ApiBundle/Tests/DependencyInjection/SyliusApiExtensionTest.php

```diff
@@ -51,6 +51,43 @@ public function it_does_not_load_swagger_integration_if_it_does_not_exists()
         $this->assertContainerBuilderNotHasService('api_platform.swagger.action.ui');
     }
 
+    /** @test */
+    public function it_loads_filter_eager_loading_extension_restricted_operations_configuration_properly(): void
+    {
+        $this->load([
+            'filter_eager_loading_extension' => [
+                'restricted_operations' => [
+                    'shop_get' => [
+                        'resources' => [
+                            'FirstResourceClass' => [],
+                            'SecondResourceClass' => false,
```

Comment:
I get why you decided on this structure instead of the path - and I totally agree, that it is a better option. However, I don't understand why did you decide to reverse the class <> operation name order. Won't it be more natural for end users to use the same structure as they are using for path definitions? 
```suggestion
                    'FirstResourceClass' => [
                        'resources' => [
                            'shop_get' => [],
                         ],
                     ],
                    'SecondResourceClass' => [
                        'resources' => [
                            'shop_get' => [],
```

Reply:
I did it that way because we can have multiple resources with the same operation name, so instead of specifying many times 'shop_get' in the configuration, we can do it only once. Then configuration is just shorter, but I agree it's less natural for end users, so I'm open to the refactor, but I think it should be rather:
```php
            'filter_eager_loading_extension' => [
                'restricted_resources' => [                   // restricted_operations -> restricted_resources
                    'FirstResourceClass' => [
                        'operations' => [                     // operations -> resources
                            'shop_get' => [],
                        ],
                    ],
                    'SecondResourceClass' => [
                        'operations' => [                     // operations -> resources
                            'shop_get' => [],
                        ],
                    ],
                ],
            ],
```
WDYT?

Reply:
Ok, I get it. But it works because of our soft rule to name things in a given way - and this is not a solid fundament for the decision IMHO. What is more, reversed structure would still benefit if someone would like to restrict several paths from one resource (but I can agree, that it is less likely than what we have now. And the biggest argument for me is the consistency with the resource structures of AP.

In conclusion, I would prefer to have it reversed.